plugins/org.jkiss.dbeaver.ui.editors.base/src/org/jkiss/dbeaver/ui/editors/object/struct/EditForeignKeyPage.java

```diff
@@ -681,6 +672,42 @@ protected void setSelectedNode(DBNDatabaseNode node) {
         }
     }
 
+    @Nullable
+    private DBPProject getOwnerProject() {
+        DBPProject ownerProject = navigatorModel.getRoot().getOwnerProject();
```

Comment:
Root node never has project.

Reply:
But that's what you wrote before my changes